Toolset/palettes/tutorial/courses/First Run/tutorials/BMI Calculator/lessons/App Lesson.txt

```diff
@@ -338,7 +356,8 @@ action
 end step
 
 step "Set Weight Label Text"
-	Set the text of the label to "Weight (kg):"
+	Set the 'Contents' property of the label to "Weight (kg):", and 
+	press ctrl + return.
```

Comment:
I'm a bit torn about how it would, in theory, be "best" to format key names.  In GitHub Markdown, you can use the `<kbd>` tag, e.g. <kbd>Ctrl</kbd>+<kbd>Return</kbd>.  In other projects' documentation, I've wrapped the keyname in `<...>` and capitalised it, e.g. \<Ctrl>+\<Return>. WDYT?